modules/markup/markdown/transform_codespan.go

```diff
@@ -49,9 +49,28 @@ func (r *HTMLRenderer) renderCodeSpan(w util.BufWriter, source []byte, n ast.Nod
 	return ast.WalkContinue, nil
 }
 
+// cssColorHandler checks if a string is a render-able CSS color value.
+// The code is from "github.com/microcosm-cc/bluemonday/css.ColorHandler", except that it doesn't handle color words like "red".
```

Comment:
This "link" is 404.

Reply:
It is not a link, it just refers to the package and its method.

And, modern IDE could resolve the method name:

![image](https://github.com/go-gitea/gitea/assets/2114189/40fdcc67-43c9-4035-b5cd-35e82cb829d5)

Reply:
> I wonder if it can handle hex RRGGBBAA syntax.

![image](https://github.com/go-gitea/gitea/assets/2114189/0b32ce56-b1d4-4218-8dbe-bec6c5e7ab5f)

Reply:
Ok. Seems far-fetched that a IDE should highlight inside "links". Better to put web links next time.

Reply:
And no, I don't mean `rgba()` syntax, bug `#rrggbbaa`.

Reply:
It's a upstream bug that it does not support `#rgba`:

https://github.com/microcosm-cc/bluemonday/blob/19901a8499025acb45db06a9cc4a5711bb4d15ac/css/handlers.go#L294

Reply:
> Ok. Seems far-fetched that a IDE should highlight inside "links". Better to put web links next time.

But Golang's package import is not a link, it is just a package path : `"github.com/microcosm-cc/bluemonday/css"`

Reply:
> And no, I don't mean `rgba()` syntax, bug `#rrggbbaa` and also `#rgba`.
> It's a upstream bug that it does not support `#rgba`:

Sorry, I don't understand what do you mean. I think `#rgba` is well-supported.

![image](https://github.com/go-gitea/gitea/assets/2114189/1cbc3e26-e306-4c25-a5c7-48ab5c2ad0a0)

![image](https://github.com/go-gitea/gitea/assets/2114189/45981e1a-147a-43c6-81d0-f58f7595f583)

Reply:
No, it only support 3,6,8 length, not 4 length. The regexp could be simplified to:

```
^#([0-9a-f]){3,4,6,8}$
```

Reply:
Opened https://github.com/microcosm-cc/bluemonday/pull/207